modules/assim.sequential/R/Remote_helpers.R

```diff
@@ -0,0 +1,384 @@
+          my_host,
+          file.path(settings$outdir,pft.path),
+          paste0(settings$host$folder,"//",folder_name,"//"),
+          delete = FALSE,
+          stderr = FALSE
+        )
+      }else{
+        PEcAn.logger::logger.severe("I don't have access to your site.pft file path !")
+      }
+    })
+
+  
+  #----------------------------------------------------------------
+  # Cleaning up the settings and getting it ready
+  #---------------------------------------------------------------
+  settings$outdir <- paste0(settings$host$folder,"/", folder_name)
+  
+  settings$host$name <- "localhost"
+  #setting the new run and out dirs
+  settings$host$rundir <- paste0(settings$host$folder,"/", folder_name,"//run")
+  settings$host$outdir <- paste0(settings$host$folder,"/", folder_name,"//out")
+  settings$rundir <- paste0(settings$host$folder,"/", folder_name,"//run")
+  settings$modeloutdir  <-paste0(settings$host$folder,"/", folder_name,"//out")
+  
+  save.setting.dir <- tempdir()
+  PEcAn.settings::write.settings(settings, basename(settingPath), save.setting.dir)
+  
+  # copying over the settings
+  remote.copy.to(
+    my_host,
+    file.path(save.setting.dir, basename(settingPath)),
+    settings$outdir,
+    delete = FALSE,
+    stderr = FALSE
+  )
+  #----------------------------------------------------------------
+  # Copying over the luncher and sending the command
+  #---------------------------------------------------------------
+  # copying over the luncher
+  remote.copy.to(
+    my_host,
+    system.file("RemoteLauncher", "SDA_launcher.R", package = "PEcAn.assim.sequential"),
+    settings$outdir,
+    delete = FALSE,
+    stderr = FALSE
+  )
+  
+  cmd <- paste0("nohup  Rscript ",
+                settings$outdir,"//SDA_launcher.R ", # remote luncher
+                settings$outdir,"//",basename(settingPath), # path to settings
+                " Obs//", basename(ObsPath), # Path to Obs
+                " > ", 
+                settings$outdir,"/SDA_remote_nohup.out 2>&1 &"
+  )
+  
+
+
+   PEcAn.logger::logger.info("Running this command on your remote: \n")
+   PEcAn.logger::logger.info(cmd)
+  
+   
+   #calling SDA
+   out<-remote.execute.R(paste0("system(\" ",cmd, "\")"),
+                         my_host,
+                         user = my_host$user,
+                         scratchdir = ".")
+   
+   
+   
+   # Let's see what is the PID of the job doing the nohup
+   # I'll use this to track the progress of my SDA job
+   PIDS<-remote.execute.cmd(my_host, cmd = "lsof",
+                      args = c(paste0(settings$outdir,"/SDA_remote_nohup.out")))
+   
+   if (length(PIDS)>1){
+     #some cleaning
+     PID<-PIDS[-1] %>% 
+       map_dbl(function(line){
+         ll <- strsplit(line, " ")[[1]]
+         ll <- ll[nchar(ll)>0]
+         ll[2] %>% 
+           as.numeric()
+       }) %>%
+       unique()
+   }else{
+     PEcAn.logger::logger.severe("Something broke the run before it starts!")
```

Comment:
Could this error message be more informative about what actually broke and where to look, or even just what actually failed to occur (e.g. "could not find remote process ID")

Reply:
Isnt using a *nohup.out file too specific or do you just want folks submitting jobs via nohup instead of say tmux?  I suppose the idea here is to submit the run as a process or to the queue?

Reply:
@serbinsh I'm not following your response to my comment.